### ISSUE TITLE
Simulator crashes in Mac OS Sierra with AMD Radeon graphics

### ISSUE THREAD

**Issue by tkandala** [udacity/sdc-issue-reports#401]

**Comment by tkandala** [comment]
I have tried to run the simulator in my macbook pro - It is a 2015 model with i7 processor, AMD Radeon R9 M370X dedicated graphics and Intel Iris Pro system graphics. It crashes immediately after hitting the play button in the first screen for any set of combinations of quality and resolution.

**Comment by awbrown90** [comment]
@tkandala do you want to try it again with this newest Mac build here https://carnd.slack.com/messages/beta-testing/ I used universal Mac build configuration setttings.

**Comment by awbrown90** [comment]
@tkandala the simulator code has now been open sourced, https://github.com/udacity/self-driving-car-sim. Can you please try loading it up on Unity and see exactly what error message you are getting?

**Comment by awbrown90** [comment]
@tkandala any new updates on this issue or it ok to close it out?

**Comment by tkandala** [comment]
Do you have the link to the latest version? last time I tried - Feb 5th, it didn't work.

**Comment by baumanab** [comment]
Closed as not actionable without updated information.

**Comment by tkandala** [comment]
Hi Aaron,

I finally found out why my program just closes upon launch without any warnings or errors. I downloaded Unity software and tried to run it and it behaved the same way. When contacted support, they asked for Editor.log file which was missing. Then I realized that my computer had permission issues where log files were not allowed to be created in my Library/Logs folder. I had the same issue with CLion although it gives me an error saying it doesn't have permissions to access the log file.
So I created a folder called Unity inside the logs folder and voila! simulator worked fine, Unity software worked fine. Back in Business!

Thanks for your support.
